.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,32 @@
-<!-- Please provide a clear and meaningful description to the CHANGELOG.md file if this PR contributes some significant changes -->
+<!--
+Thank you for your Pull Request. Please provide a description above and review
+the requirements below.
+
+Bug fixes and new features should include tests and possibly benchmarks.
+
+Contributors guide: https://github.com/theia-ide/theia/blob/master/CONTRIBUTING.md
+-->
+
+#### What it does
+<!-- Include relevant issues and describe how they are addressed -->
+
+#### How to test
+<!-- Describe how a reviewer can verify changes within Theia repo -->
+
+#### Review Checklist
+<!-- For completed items, change [ ] to [x].  -->
+<!-- For inapplicable items, cross out with `[ ] foo` to `[x] ~~foo~~`.  -->
+
+- [ ] commits are signed-off: https://github.com/theia-ide/theia/blob/master/CONTRIBUTING.md#sign-your-work
+- [ ] commit history is cleaned up of merge and fix up commits
+  - use `git pull -r` to pick up changes from the master
+- [ ] tests/benchmarks are defined for functionallity/perf optimizations and verified by a reviewer, not an author
+- [ ] [changelog](https://github.com/theia-ide/theia/blob/master/CHANGELOG.md) is updated
+- [ ] breaking changes are justified and recorded in the changelog
+- [ ] each new file has a proper copyright with the current year and an author as an entity
```

Comment:
each new file has a proper copyright with the current year and the name of contributing entity (individual or company)